src/pharmpy/random_variables.py

```diff
@@ -1185,7 +1185,10 @@ def _sample_expr_from_rvs(sampling_rvs, expr, parameters, samples, rng):
 
     ordered_symbols, fn = _lambdify_canonical(expr)
     data = _sample_rvs_subset(sampling_rvs, ordered_symbols, samples, rng)
-    return fn(*data)
+    results = fn(*data)
+    # FIXME Why is this necessary? Was not before sympy 1.11.0 and is not with
+    # cse=False.
+    return results[0] if isinstance(results, list) else results
```

Comment:
See https://github.com/sympy/sympy/pull/23538#issuecomment-1225962565

Reply:
https://github.com/sympy/sympy/pull/23975 was merged and [the patch is included in sympy 1.11.1](https://github.com/sympy/sympy/wiki/release-notes-for-1.11.1#changes). I'll rebase, edit, and resolve later.

Reply:
Reverted since the bug was fixed upstream.